deltachat-ios/Controller/QrViewController.swift

```diff
@@ -318,3 +240,81 @@ class QrViewController: UITableViewController, QrCodeReaderDelegate {
         navigationController?.pushViewController(chatVC, animated: true)
     }
 }
+
+// MARK: - QRCodeDelegate
+extension QrViewController: QrCodeReaderDelegate {
```

Comment:
out of curiosity: this is a style question to move delegate implementations to the extension, right?

Reply:
Technically it makes no difference, so yes, it's a style question.

IMO it improves readability by separating a class after concerns. And if you use "MARK-statements" you can find them very easily. 

If you decide later to extract the delegate to a different class (for example if you want the class' viewModel to implement the delegate instead of the class), you can do so by basicly changing one word (`extension ViewModel: QRCodeReaderDelegate`).